pkg/commands/cancel.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 Praetorian Security, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/praetorian-inc/trident/pkg/db"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"net/http"
+)
+
+var cancelCommand = &cobra.Command{
+	Use:   "cancel",
+	Short: "cancel campaign execution",
+	Long:  `can be used to halt a running campaign and stop all further spraying.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		cancelPost(cmd, args)
+	},
+}
+
+func init() {
+	cancelCommand.Flags().StringVarP(&campaignID, "campaign", "c", "1",
```

Comment:
Should this default to anything given the operation?

Reply:
👍 changing to no default - we don't want to accidentally cancel something.